lib/llm/src/model_card/create.rs

```diff
@@ -135,6 +122,7 @@ impl ModelDeploymentCard {
             service_name: model_name.to_string(),
             model_info: Some(ModelInfoType::from_repo(repo_id).await?),
             tokenizer: Some(TokenizerKind::from_repo(repo_id).await?),
+            gen_config: GenerationConfig::from_repo(repo_id).await.ok(), // optional
```

Comment:
I think it would be helpful to give an example model/arch or two for each thing here for some context on why we need them all. ex:
```
  // Needed by Gemma3 models
  gen_config: ...
```